### PR TITLE
Deviceflow fix

### PR DESCRIPTION
**Pull request by himeshsiriwardana**

### CODE REVIEW

**Review by janakamarasena — APPROVED** [review]

File: en/docs/guides/access-delegation/try-device-flow.md [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
+specification.
 
 !!! info
-    For more information, see [Device Authorization Grant]({{base_path}}/references/concepts/authorization/device-flow-grant). 
+    For more information, see [Device Authorization Grant]({{base_path}}/references/concepts/authorization/device-flow-grant).
     
 Follow the topics given below to try out the Device Authorization grant type.
 
-## Set up WSO2 Identity Server
+## Configure WSO2 Identity Server
```
Comment (nilminiwso2) [inline en/docs/guides/access-delegation/try-device-flow.md:11]:
```suggestion
## (Optional) Configure WSO2 Identity Server
```

File: en/docs/guides/access-delegation/try-device-flow.md [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
+
+!!! info
+    **This step is optional** unless it is required to customize the properties below.
```
Comment (nilminiwso2) [inline en/docs/guides/access-delegation/try-device-flow.md:14]: Are we saying this?

`If you want to change the default values for the following parameters, add the configuration to the `deployment.toml` file with the required values.`

If so, should the user know the default values the effective in the system?

Or are we saying the following?

`If required, you can enforce the following parameters by adding the configuration to the `deployment.toml` file.`

In this case, there are no default values.

Comment (nilminiwso2) [reply en/docs/guides/access-delegation/try-device-flow.md:14]: @janakamarasena

Comment (janakamarasena) [reply en/docs/guides/access-delegation/try-device-flow.md:14]: We have default values. IIRC what we have put as a sample configuration are the default values(@chamathns). We can display them in the doc saying these are the default.

Comment (chamathns) [reply en/docs/guides/access-delegation/try-device-flow.md:14]: yes, these are the default values. The user does not need to add any configs to the toml file to try device flow. It should work ootb with the default values. we should word the documentation in a way that does not imply it is required add those configs to the deployment.toml

**Pull request merged**